src/pdf_beamtime/src/pdf_beamtime_server.cpp

```diff
@@ -16,16 +16,19 @@ PdfBeamtimeServer::PdfBeamtimeServer(
   // Add the obstacles
   planning_scene_interface_.applyCollisionObjects(create_env());
 
-  // Create the services
+
+  // // Create the services
   new_box_obstacle_service_ = node_->create_service<BoxObstacleMsg>(
     "pdf_new_box_obstacle",
     std::bind(
-      &PdfBeamtimeServer::new_box_obstacle_service_cb, this, _1, _2));
+      &PdfBeamtimeServer::new_obstacle_service_cb<BoxObstacleMsg::Request,
+      BoxObstacleMsg::Response>, this, _1, _2))
```

Comment:
I think this is missing a semi-colon. Should solve the uncrustify failure. 
```suggestion
      BoxObstacleMsg::Response>, this, _1, _2));
```

Reply:
missing semi-colon is added. 
https://github.com/maffettone/erobs/blob/82ea0b0722451076fc8c67563b396d2862af8a5a/src/pdf_beamtime/src/pdf_beamtime_server.cpp#L24